.github/workflows/kind.yml

```diff
@@ -50,8 +50,10 @@ jobs:
 
     - name: Verify format
       run: |
+        make fmt-dependencies
```

Comment:
If the `fmt` target requires `fmt-dependencies`, is this necessary? At least, I would want it to be consistent here with the `lint` and `lint-dependencies` targets.

Reply:
You're right. Thanks, I'll add this to a followup PR.